pkg/appliance/functions.go

```diff
@@ -776,3 +788,251 @@ func PrettyBytes(v float64) string {
 	}
 	return fmt.Sprintf("%.2fYB", v)
 }
+
+type ManifestConfig struct {
+	MediaType string
+	Size      int
+	Digest    string
+}
+type DockerManifest struct {
+	SchemaVersion int
+	MediaType     string
+	Config        ManifestConfig
+	Layers        []ManifestConfig
+}
+
+type ImageJSON struct {
+	Image string `json:"image"`
+}
+
+func DownloadDockerBundles(ctx context.Context, out io.Writer, client *http.Client, zipName, registry string, images map[string]string, ciMode bool) (*os.File, fs.FileInfo, error) {
+	// Create zip-archive
+	archive, err := os.Create(fmt.Sprintf("/tmp/%s.zip", zipName))
```

Comment:
this wont work on windows, use https://pkg.go.dev/os#CreateTemp instead

Reply:
:+1: